_data/header.yml

```diff
@@ -1,9 +1,9 @@
 ---
 title: Solidity Summit
-date: 29+30 April 2020<br>Berlin
+date: 29+30 April 2020<br>Online
```

Comment:
"Virtual"?

Reply:
Good idea. Maybe we can say "virtual" there and then change the text in red (the sub heading) to say `Two-day, friendly-sized, interactive online event with talks and workshops`

Reply:
Sounds good.

Still wondering how it will be done though :wink: